src/app-layout/visual-refresh/drawers.scss

```
@@ -156,7 +156,7 @@
 
   > .drawer-slider {
     grid-column: 1;
-    grid-row: 3;
+    grid-row: 1 / span 3;
```

nit: I'm not sure if the number of rows might change, but we could be a bit more defensive here.
```suggestion
    grid-row: 1 / -1;
```

---

This is a great solution! There are a couple other places in this file that use the `span` method, so they should all be consistent. I think this works for now, since the rows won't change.